tests/format.test.js

```diff
@@ -49,9 +55,44 @@ let javascript = [
     `;<div class="block px-1\u3000py-2" />`,
     `;<div class="px-1\u3000py-2 block" />`,
   ],
+
+  // Whitespace is normalized and duplicates are removed
+  [
+    ';<div class="   m-0  sm:p-0  p-0   " />',
+    ';<div class="m-0 p-0 sm:p-0" />',
+  ],
+  [
+    ";<div class={'   m-0  sm:p-0  p-0   '} />",
+    ";<div class={'m-0 p-0 sm:p-0'} />",
+  ],
+  [';<div class={` sm:p-0\n  p-0   `} />', ';<div class={`p-0 sm:p-0`} />'],
+  [';<div class="flex flex" />', ';<div class="flex" />'],
+  [';<div class={`   flex  flex `} />', ';<div class={`flex`} />'],
+  [
+    ';<div class={`   flex  flex flex${someVar}block block`} />',
+    ';<div class={`flex flex${someVar}block block`} />',
+  ],
+  [
+    // This happens because we we look at class lists individually but
+    // a future improvement could be to dectect this case and not
+    // remove the space after flex.
+    ';<div class={`flex ` + `text-red-500`} />',
+    ';<div class={`flex` + `text-red-500`} />',
+  ],
+  [
+    // This happens because we we look at class lists individually but
+    // a future improvement could be to dectect this case and not
+    // remove the space after flex.
```

Comment:
@thecrypticace Noticed this comment is a duplicate of the comment above and isn't relevant for this test case 👍

Reply:
Ah yep — this comment is gone with the latest updates but good catch. 👍